patches/minecraft/net/minecraft/world/level/newbiome/layer/BiomeEdgeLayer.java.patch

```diff
@@ -0,0 +1,15 @@
+--- a/net/minecraft/world/level/newbiome/layer/BiomeEdgeLayer.java
++++ b/net/minecraft/world/level/newbiome/layer/BiomeEdgeLayer.java
+@@ -19,7 +_,11 @@
+                   return 23;
+                }
+             }
+-
++            net.minecraftforge.registries.ForgeRegistry<net.minecraft.world.level.biome.Biome> registry = (net.minecraftforge.registries.ForgeRegistry<net.minecraft.world.level.biome.Biome>)net.minecraftforge.registries.ForgeRegistries.BIOMES;
++            java.util.Optional<net.minecraft.resources.ResourceKey<net.minecraft.world.level.biome.Biome>> edge = net.minecraftforge.common.BiomeManager.getEdgeBiome(registry.getKey(p_76669_), registry.getKey(p_76665_), registry.getKey(p_76666_), registry.getKey(p_76667_), registry.getKey(p_76668_));
```

Comment:
Is there a reason you are getting the registry here rather than just passing the parameters to the `getEdgeBiome` method and then grabbing the registry and looking up parts of the registry based on those passed in parameters there?

Reply:
Not really, I just felt like it was cleaner to keep the integer ids away from the BiomeManager. But I can change that.